doc/lua_api.md

```diff
@@ -9903,22 +9644,22 @@ degradation in older clients).
 
 ```lua
 {
-  -- old syntax
-  maxpos = {x = 0, y = 0, z = 0},
-  minpos = {x = 0, y = 0, z = 0},
-
-  -- absolute value
-  pos = 0,
-  -- all components of every particle's position vector will be set to this
-  -- value
-
-  -- vec3
-  pos = vector.new(0,0,0),
-  -- all particles will appear at this exact position throughout the lifetime
-  -- of the particlespawner
-
-  -- vec3 range
-  pos = {
+    -- old syntax
```

Comment:
so now this is 4 spaces?
shouldn't we be using tabs?

Reply:
The rest of the Lua codeblocks use spaces instead of tabs. I'll open another PR for that in the future.